tests/cli/test_datasets.py

```diff
@@ -622,3 +622,49 @@ def test_dataset_rm(tmpdir, runner, project, client):
 
     result = runner.invoke(cli.cli, ['doctor'], catch_exceptions=False)
     assert 0 == result.exit_code
+
+
+@pytest.mark.integration
+def test_dataset_import_real_doi(runner, project):
+    """Test dataset import for existing DOI."""
+    result = runner.invoke(
+        cli.cli, ['dataset', 'import', '10.5281/zenodo.597964'], input='y'
```

Comment:
make sure the tests won't contact zenodo or any external service (see also #470)

Reply:
these are still contacting zenodo.org, no? can you mock the responses?

Reply:
We discussed this. :) 

Mocking this will make things very hard to see when they update their API in the future. I marked those integration tests as integration and the rest of the tests which #470 is referring to needs to get marked as well. Then we will put into py.test runner by mark and separate unit from integration tests on the runner level.

Reply:
@jirikuncar I tend to agree -- their API doesn't seem to be versioned so actually querying the API in the tests makes sure we know that we can still use it. However, I also agree that we shouldn't depend on the connection/server to be up for the tests to pass. Can we have a mocked version of the test as a unit test and the "live" version to test compatibility with zenodo.org?